content/docs/installation/best-practice.md

```diff
@@ -22,6 +22,201 @@ are designed for backwards compatibility rather than for best practice or maximu
 You may find that the default resources do not comply with the security policy on your Kubernetes cluster
 and in that case you can modify the installation configuration using Helm chart values to override the defaults.
 
+## High Availability
+
+cert-manager has three long-running components: controller, cainjector, and webhook.
+Each of these components has a Deployment and by default each Deployment has 1 replica
+but this does not provide high availability.
+The Helm chart for cert-manager has parameters to configure the `replicaCount` for each Deployment.
+In production we recommend the following `replicaCount` parameters:
+
+```yaml
+replicaCount: 2
+webhook:
+  replicaCount: 3
```

Comment:
I think a user might want even more webhook replicas, maybe we should explain why we set the replicas for leader-elected controllers to 2 and for non-leader elected controllers to >= 3?